docs/Web-Flow-Configuration.md

```diff
@@ -67,7 +67,35 @@ powerauth.webflow.service.applicationEnvironment=
 
 # Configuration of Offline mode
 powerauth.webflow.offlineMode.available=true
+
+# Configuration of Android Security Warning
+powerauth.webflow.android.showSecurityWarning=true
+```
+
+Encryption of user passwords during transport can be configured using following properties:
 ```
+# Configuration of Password Encryption
+powerauth.webflow.authentication.type=PASSWORD_ENCRYPTION_AES
```

Comment:
I am not sure about this property naming - I see why is this later in the code, but maybe having:

```
powerauth.webflow.password.encryption.type=...
```

... would make better sense from configuration perspective?

Reply:
We can change it, no problem.